### ISSUE TITLE
[Bug] Can't uniformly lower brightness of website

### ISSUE THREAD

**Issue by bluelightzero** [darkreader/darkreader#12154]
### Prerequisites

- [X] I [searched for any existing report](https://github.com/darkreader/darkreader/issues?q=is%3Aissue) about this bug to avoid opening a duplicate.
- [X] I can reproduce this bug in a new, unmodified web browser profile with Dark Reader installed as the only extension.
- [X] I understand I need to use the [Broken Website Report template](https://github.com/darkreader/darkreader/issues/new?assignees=&labels=Broken+Website&template=broken-website-report.yml&title=%5BBroken+Website%5D+Replace+with+title) if this bug I am reporting occurs on a single website.

### Bug Description

At some point, it was possible to set **_only the brightness_** and have it apply to the whole page uniformly without trying to filter certain elements.

I have since completely reinstalled all my plugins and started over and can no longer achieve the same affect.

### Website Address

https://forum.godotengine.org/t/how-can-i-add-a-heightmap-to-a-meshtexture/10930

### Steps To Reproduce

Here are the settings I use:

![image](https://github.com/darkreader/darkreader/assets/4492982/4316e60e-a352-4c33-ad8b-0fc956f2ebb3)

### Expected Behavior

This is the effect I want to achieve (When brightness setting is lowered):

![image](https://github.com/darkreader/darkreader/assets/4492982/18a65de6-b5c9-4dd4-aa35-1cb93c17c459)

### Actual Behavior

Here is what each filter looks like:

None: 

![image](https://github.com/darkreader/darkreader/assets/4492982/0ff94ab1-4f2a-46a3-a512-d33b7bb89e8a)

Filter:

![image](https://github.com/darkreader/darkreader/assets/4492982/ae3aa54c-bef1-4ee2-a795-e5c3df8e3bd8)

Filter+:

![image](https://github.com/darkreader/darkreader/assets/4492982/e0345563-3070-46b6-89de-23d1dd740f29)

Static:

![image](https://github.com/darkreader/darkreader/assets/4492982/53919cb8-b25f-4588-b077-97a910b21f1b)

Dynamic:

![image](https://github.com/darkreader/darkreader/assets/4492982/8e5c6320-7448-46ea-83ac-f23532332ef2)

### Operating System

Fedora 39

### Web Browser name and version

Firefox 121.0

### Dark Reader version

4.9.74

**Comment by Myshor** [comment]
First of all... bug is not related to operating system and browser.
I replicated it using Firefox in Windows 11 and Edge in Windows 11.

Second thing - bug is not related to website. I checked other sites with images and it's the same.
Dark reader set brightness filter to text and SVG but not images.

Is it a bug? For me it's hard to say. I never tried to make pictures darker using sliders in Dark Reader.

Let @alexanderby decide if it's a bug or a feature that Dark reader doesn't touch images. 😉

**Comment by bluelightzero** [comment]
Yeah, I wasn't sure. I just gave a website as an example that I knew had the issue.

I thought one of the filter types was supposed to uniformly apply to everything. At least it did work like this at some point.

See, by default I prefer websites just being less bright, it has the same effect as turning the monitor brightness down, but sometimes I will setup specific rules for certain websites.
